README.md

```diff
@@ -29,9 +29,16 @@ CHT Sync has been specifically designed to work in both local development enviro
 - `Docker`
 - An `.env` file containing the environment variable placeholders from the `.env.template` file. The file should be located in the root directory of the project or set by the operating system. The variables should be customized accordingly for the specific deployment needs.
 
+#### Environment variable
+We have four environment variable groups in the `.env.template` file. It is important to understand the difference between them.
```

Comment:
```suggestion
There are four environment variable groups in the `.env.template` file. To successfully set up `cht-sync`, It is important to understand the difference between them.
```

Reply:
Added a clarification on why it is important to know the difference between the variables, and removed the "we" mention.